mp2-test/src/mpt_sequential.rs

```diff
@@ -111,7 +112,7 @@ pub fn generate_receipt_proofs() -> Vec<ReceiptProofInfo> {
     rt.block_on(async {
         // Spin up a local node.
 
-        let rpc = ProviderBuilder::new().on_anvil();
+        let rpc = ProviderBuilder::new().on_anvil_with_config(|anvil| Anvil::block_time(anvil, 1));
```

Comment:
mhhh why are you introducing time effects now ? I don't understand the reason. Is it to have multiple tx inside a block or stg ? 
Couldn't we use [multicalls](https://crates.io/crates/alloy-multicall) instead for example ?